index.html

```diff
@@ -36,13 +36,20 @@ <h2>サービス内容</h2>
         </li>
       </ul>
 
-      <h2>スピードショップとは誰のことですか?</h2>
+      <h2>メンバー</h2>
```

Comment:
The original title in English is "Who's speedshop?", and the direct translation of it in Japanese sounded a little weird. So I changed it to just '"Members" in Japanese. I think "Staff" is also another common term for this. Let me know what you like.